packages/inventory-compliance/src/Compliance.js

```diff
@@ -57,12 +59,21 @@ query System($systemId: String!){
 }
 `;
 
+const columns = [
```

Comment:
It wouldn't be the worst to export this to use in the tests, but just a suggestion.